2025-03-31 20:33:19.060 [warning] Dir "/data/tmp/rm-test-async/layers/api/.pixi/envs" is not watchable (directory does not exist)
2025-03-31 20:33:19.060 [info] Python interpreter path: ./venv/bin/python
2025-03-31 20:33:19.060 [info] > pyenv which python
2025-03-31 20:33:19.060 [info] cwd: .
2025-03-31 20:33:21.208 [info] Starting Pylance language server.
2025-03-31 20:33:59.426 [info] > conda info --json
2025-03-31 20:33:59.611 [info] > ~/.conda/bin/conda info --json
2025-03-31 20:33:59.631 [info] > ~/miniconda3/bin/conda info --json
2025-03-31 20:34:00.756 [info] > . ./venv/bin/activate && echo 'e8b39361-0157-4923-80e1-22d70d46dee6' && python ~/.vscode-oss/extensions/ms-python.python-2025.2.0-linux-x64/python_files/printEnvVariables.py
2025-03-31 20:34:00.756 [info] shell: bash
2025-03-31 20:34:00.776 [info] > . ./venv/bin/activate && echo 'e8b39361-0157-4923-80e1-22d70d46dee6' && python ~/.vscode-oss/extensions/ms-python.python-2025.2.0-linux-x64/python_files/printEnvVariables.py
2025-03-31 20:34:00.776 [info] shell: bash
